### ISSUE TITLE
Config for LIS dataset

### ISSUE THREAD

**Issue by PhanMinhPhong2422003** [Linwei-Chen/LIS#21]
I want to reproduce the result which only LIS dataset is used, not COCO. But i cannot find any config file for it. Could you help me with it?

**Comment by Linwei-Chen** [comment]
Thanks for your interest. You can refer to https://github.com/Linwei-Chen/LIS/blob/main/Configs/mask_rcnn_r50_fpn_caffe_AWD_SCB_DSL_LIS2LIS.py

**Comment by PhanMinhPhong2422003** [comment]
> Thanks for your interest. You can refer to https://github.com/Linwei-Chen/LIS/blob/main/Configs/mask_rcnn_r50_fpn_caffe_AWD_SCB_DSL_LIS2LIS.py

I have tried to make the config for train and test both on sRGB-dark LIS dataset only, and base on your annotations, using Mask _RCNN and Resnet50 as backbone, and with the basic config of mmdet. However, my results is not as high as your in the table2 in the paper . My result are : AP is 30,4 and APbox is 35.4 which in your paper is 35.5 and  42.9 respectively.
 I don't know if is there any special  finetune or  have i misunderstanding the result in the paper, could you show me the config of it. 

Additionaly, when i tried to train and test the same model with Raw-dark, my result is only 31.2 and 36. which is much more lower than your

Thank you for your help!

![Image](https://github.com/user-attachments/assets/335fbe49-6b7f-49ad-a485-aa1509281bc0)

**Comment by Linwei-Chen** [comment]
Thanks for your interest. Note that all results are based on the COCO Pretrained model (3x from mmdet, followed by pretraining on COCO-8-cls for 1x).

**Comment by PhanMinhPhong2422003** [comment]
> Thanks for your interest. Note that all results are based on the COCO Pretrained model (3x from mmdet, followed by pretraining on COCO-8-cls for 1x).


Sorry , i am just a beginer in this field. And to be honest,  i do not understand your answer, and how to do it.
Could you provide more details about how to do it?
(I don't understand what it mean " 3x from mmdet" , is it mean the schedule ? And i have to dowload the COCO dataset and train on it? 
And how to followed pretrining on COCO-8-cls for 1x? it mean that after i train on coco dataset, i load the checkpoint from the  before stage, then  what should i do next,  i mean should i train on coco-8 dataset ? 
And after that i shoud load it as the checkpoint, or the init config at the backbone of resnet? 
Or if it have specific pretrain on the internet, which one should i use? )

Thank you very much  for your explanation!

**Comment by Linwei-Chen** [comment]
https://download.openmmlab.com/mmdetection/v2.0/mask_rcnn/mask_rcnn_r50_caffe_fpn_mstrain-poly_3x_coco/mask_rcnn_r50_caffe_fpn_mstrain-poly_3x_coco_bbox_mAP-0.408__segm_mAP-0.37_20200504_163245-42aa3d00.pth

https://github.com/open-mmlab/mmdetection/tree/main/configs/mask_rcnn

You can refer to links above to get the 3x (36 epoch) Pretrained Model
1x means standard 12 epoch training setting.
